src/jefe_recambios.go

```diff
@@ -0,0 +1,61 @@
+package src
+
+import (
+	"sort"
+	"time"
+)
+
+type JefeRecambios struct {
+	pedidosPendientes       []Pedido
+	repartidoresDisponibles []Repartidor
+	repartosActivos         map[*Pedido]map[*Repartidor]*HoraEntrega //Ojo con esto que son punteros para que sean comparables.
+}
+
+func NuevoJefeRecambios() *JefeRecambios {
+	return &JefeRecambios{}
+}
+
+func (j *JefeRecambios) NUEVOPEDIDO(p Pedido) {
+	j.pedidosPendientes = append(j.pedidosPendientes, p)
+
+	// Ordenar la lista de pedidos por tiempo estimado
+	sort.Slice(j.pedidosPendientes, func(i, h int) bool {
+		return j.pedidosPendientes[i].UBICACION().tiempoEstimadoMinutos > j.pedidosPendientes[h].UBICACION().tiempoEstimadoMinutos
```

Comment:
Aquí pasa como te he comentado más abajo, al ser parte de la lógica de negocio de otro hito, no creo que haga falta establecer una estimación del tiempo de entrega por ahora.

Reply:
Si te fijas en los issues correspondientes, al desgranar la HU salieron esas funcionalidades. Si no las quieres por ahora las elimino, pero no se si estará correcto entonces. Lo que me digas, espero tu respuesta. Un saludo

Reply:
Pues podemos probar a dejarlo entonces y que nos lo corrijan, y ya vemos qué hacemos con después 👍.

Reply:
Debéis resolver todos los issues antes de proponerlo para revisión.

Reply:
Y si queréis dejarlo copiadlo a otra rama y borradlo de aquí. Sólo se debe hacer lo que pida el PMV correspondiente.

Reply:
Solucionado en [5fe682a](https://github.com/johnwaves/recambios-express/pull/10/commits/5fe682a597e4d02dce7adfc2b0f155bd893f6bb7)